### ISSUE TITLE
Page subtitle doesn't have ARIA

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-core#244]
### Subject of the issue/enhancement/features
`.page__subtitle` isn't distinguishable as a title. For accessibility, all titles should be identified as a heading and include an appropriate aria-level for the hierarchy of the title. 

### Expected behaviour
For consistency with `.page__title` I would expect the on screen title to have `aria-hidden="true"` and the subtitle text to be included in the `.js-heading` aria-label. The subtitle could have it's own aria-label but I think combining the page title and subtitle in one aria-label under a single aria-level is sufficient but happy for folks to tell me otherwise?

For example:

```
      {{#if displayTitle}}
      <div class="page__title">

        {{#unless _disableAccessibilityState}}
        <div class="js-heading">INCLUDE BOTH TITLE AND SUBTITLE</div>
        {{~/unless~}}

        <div class="page__title-inner" {{#unless _disableAccessibilityState}}aria-hidden="true"{{/unless}}>
          {{{compile displayTitle}}}
        </div>

      </div>
      {{/if}}

      {{#if subtitle}}
      <div class="page__subtitle">
        <div class="page__subtitle-inner {{#unless _disableAccessibilityState}}aria-hidden="true"{{/unless}}">
          {{{compile subtitle}}}
        </div>
      </div>
      {{/if}}
```

**Comment by oliverfoster** [comment]
Connected to https://github.com/adaptlearning/adapt-contrib-core/issues/235
